app/routes/questionnaire.py

```diff
@@ -180,7 +166,6 @@ def get_section(schema, questionnaire_store, section_id, list_item_id=None):
 @questionnaire_blueprint.route(
     "<list_name>/<list_item_id>/<block_id>/", methods=["GET", "POST"]
 )
-@login_required
```

Comment:
There are still a few methods with unnecessary `login_required` decorators now that we do it on the blueprint.